tests/test_client.py

```diff
@@ -294,5 +300,8 @@ def test_upload_annotations():
 @responses.activate
 def test_train_detector():
     add_mock_detector_train_responses(1)
+    add_mock_operations_responses('pending')
```

Comment:
IIRC, we don't expose pending in the public API: we only have running, success and failed

Reply:
The current `/operations/` endpoint returns `job.status` (so `pending` can be returned). I agree that might have no value to the user (plus, if they see it `pending`, users might wrongly believe an action is needed on their part). I'll fix that in the public api.

Reply:
https://gitlab.com/picterra/platform/-/merge_requests/2241

Reply:
Ah I thought we already did that. Thanks !